tests/rules/relative-font-units.test.js

```diff
@@ -0,0 +1,723 @@
+/**
+ * @fileoverview Tests for relative-font-units rule.
+ * @author Tanuj Kanti
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/relative-font-units.js";
+import css from "../../src/index.js";
+import { RuleTester } from "eslint";
+import dedent from "dedent";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		css,
+	},
+	language: "css/css",
+});
+
+ruleTester.run("relative-font-units", rule, {
+	valid: [
+		"a { font-size: 1rem; }",
+		"a { font: 2rem Arial, sans-serif; }",
+		"a { font: 1.2rem/2 Arial, sans-serif; }",
```

Comment:
Let's add test cases for all possible syntax for the `font` shorthand. https://developer.mozilla.org/en-US/docs/Web/CSS/font#syntax